### DISCUSSION TITLE
`nvim` command does not work after installation

### DISCUSSION THREAD

**Discussion by imhalid** [LazyVim/LazyVim#2674] [Q&A]
I'm pretty new here, I installed lazyvim but the `nvim` command doesn't work. I have my nvim folder installed in `.config` file but I don't understand what the problem is. do I need to edit my `.zshrc` file?

`export PATH="$PATH:$HOME/.config/"` I added this but it doesn't work

**Comment by folke** [comment]
You need to install neovim. LazyVim is just a config. It requires you to have Neovim installed.

https://www.lazyvim.org/#%EF%B8%8F-requirements
